src/pages/en/read/[...slug].astro

```diff
@@ -13,17 +16,71 @@ if (entry === undefined) {
     return Astro.redirect("/404");
 }
 
-const author = await getEntry(entry.data.author);
+const author = (await getCollection("author")).find(a => a.id === entry.data.author.id)?.data as Author;
+
+async function getRelatedArticles() {
+    const allEntries = await getCollection("read");
+        return Promise.all(allEntries.filter((e) => e.id !== entry?.id).map(async (e) => ({
+            item: e,
+            author: (await getCollection("author")).find(a => a.id === e.data.author.id)?.data as Author,
+        })));
+}
 
-const { Content } = await entry.render();
 
 ---
 <Layout title="Read · TEDxKMUTT">
-<Content />
-{
-    author && (
-        <p>Written by: {author.data.name}</p>
-        <img src={author.data.avatar.src} alt={author.data.avatar.alt}>
-    )
-}
+
+    <main class="mx-auto max-w-lg lg:max-w-4xl pt-8">
+        <img src={entry.data.thumbnail.src} alt={entry.data.thumbnail.alt} class="object-cover rounded-xl w-full">
+        <h1 class="text-4xl font-bold mt-8">{entry.data.title}</h1>
+        <div class="flex items-center">
+            {
+                entry.data.categories.map((category) => (
+                    <span class="bg-[#46000C] mt-6 mb-2 text-red-600 rounded-full text-sm px-3 py-1 mr-2">
+                        {category.toUpperCase()}
+                    </span>
+                ))
+            }
+        </div>
+
+        <div class="pt-8">{flattenMarkdown(entry.body)}</div>
```

Comment:
Use `Content` tag (https://docs.astro.build/en/guides/content-collections/#rendering-content-to-html)

```
const { Content } = await entry.render();

---

<Content />
```